src/Flash.php

```diff
@@ -89,21 +85,35 @@ public function add(string $key, $value = true, bool $removeAfterAccess = true):
         $this->save($flashes);
     }
 
-    public function remove(string $key)
+    public function remove(string $key, $defaultValue = null)
```

Comment:
`remove()` with `$defaultValue` parameter = WTF! :P

Reply:
That is to match `get()`. Remove returns flash message.

Reply:
This is optimization for reading value. This means that either this method is badly named (the main purpose of it is to get value - removing value from collection is only additional side effect) or default value logic does not belong here and will be only confusing.

IMO it is both of them. I never liked `ArrayHelper::remove()` for exact the same reason - it is confusing to use `remove()` method for reading value from config array. I'm always trying to extend this helper with more adequate name, which will also imply getting value (like `pop()`).
And since `??` operator supporting default value is mostly pointless - `??` is more readable and more efficient.